### PR TITLE
Mon 20340 fix promote 2210

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Fix various issues and diffs on promote action.

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by kduret — APPROVED** [review]

File: .github/actions/promote-to-stable/action.yml [hunk 1]
```diff
@@ -70,32 +70,75 @@ runs:
...
+          SRC_PATHS=$(jf rt search --include-dirs rpm-standard/${{ inputs.major_version }}/${{ inputs.distrib }}/testing/$ARCH/${{ inputs.module }}/*.rpm | jq -r '.[].path')
           if [[ ${SRC_PATHS[@]} ]]; then
             for SRC_PATH in ${SRC_PATHS[@]}; do
               echo "[DEBUG] - Source path found: $SRC_PATH"
             done
           else
-            echo "[DEBUG] - No source path found."
-            continue
+            echo "[DEBUG] - No source path found for $ARCH."
+            PATHS_FOUND_IN_ARCH+=1
```
Comment (sc979) [inline .github/actions/promote-to-stable/action.yml:84]: Source path not found +1

File: .github/actions/promote-to-stable/action.yml [hunk 2]
```diff
@@ -70,32 +70,75 @@ runs:
...
           done
         done
       shell: bash
 
-    - name: Promote RPM packages to stable on yum.centreon.com
+    - name: Promote DEB packages to stable
+      if: ${{ startsWith(inputs.distrib, 'bullseye') }}
+      run: |
+        set -x
+        echo "[DEBUG] - Major version: ${{ inputs.major_version }}"
```
Comment (sc979) [inline .github/actions/promote-to-stable/action.yml:107]: Seems more INFO than DEBUG

File: .github/actions/promote-to-stable/action.yml [hunk 3]
```diff
@@ -133,8 +178,13 @@ runs:
               echo "[DEBUG] - Source path found: $SRC_PATH"
             done
           else
-            echo "[DEBUG] - No source path found."
-            continue
+            echo "[DEBUG] - No source path found for $ARCH."
+            PATHS_FOUND_IN_ARCH+=1
```
Comment (sc979) [inline .github/actions/promote-to-stable/action.yml:182]: same

File: .github/actions/promote-to-stable/action.yml [hunk 4]
```diff
@@ -70,32 +70,75 @@ runs:
...
-
+          SRC_PATHS=$(jf rt search --include-dirs rpm-standard/${{ inputs.major_version }}/${{ inputs.distrib }}/testing/$ARCH/${{ inputs.module }}/*.rpm | jq -r '.[].path')
           if [[ ${SRC_PATHS[@]} ]]; then
             for SRC_PATH in ${SRC_PATHS[@]}; do
               echo "[DEBUG] - Source path found: $SRC_PATH"
             done
           else
-            echo "[DEBUG] - No source path found."
-            continue
+            echo "[DEBUG] - No source path found for $ARCH."
```
Comment (sc979) [inline .github/actions/promote-to-stable/action.yml:83]: No source path foudn <> PATHS_FOUND_IN_ARCH +1

File: .github/actions/promote-to-stable/action.yml (hunk 1 above)
Comment (sc979) [inline .github/actions/promote-to-stable/action.yml:84]: No source path found <> PATHS_FOUND_IN_ARCH +1

**Review by sc979 — APPROVED** [review]

**Pull request merged**